Add command to query GP logs to SLS integration (#38876)
demisto/content: Packs/CortexDataLake/ReleaseNotes/1_4_18.md

#### Integrations

##### Strata Logging Service XSOAR Connector

> - Added a new command **cdl-query-gp-logs** to query GlobalProtect VPN logs.

Reviewer: ```suggestion
- Added the command ***cdl-query-gp-logs*** command to query GlobalProtect VPN logs.
```
